host/4/python-build.yml

```diff
@@ -32,18 +33,27 @@ jobs:
         _37:
           pythonVersion: "3.7"
           pythonImageNameSuffix: "37"
+          dockerfilePath: "bullseye"
```

Comment:
ensure all the publish pipelines are updated as well; Use https://github.com/Azure/azure-functions-docker/pull/1108 as reference